MetaMask/metamask-extension
# Move from @download/blockies to blo

## Description

Move from @download/blockies to [blo](https://github.com/bpierre/blo), for a faster and more precise rendering of the blockies identicons, while also removing the need for the library to access `document`.

## **Manual testing steps**

1. `yarn && yarn start`
2. In Settings => General, change the identicon style to “Blockies”
3. Check everywhere there is an identicon in the extension.

## **Screenshots/Recordings**

Before:
<img src=https://github.com/MetaMask/metamask-extension/assets/36158/e4a91c97-87bc-45fc-8f08-9514bb6dca8f width=400>

After:
<img src=https://github.com/MetaMask/metamask-extension/assets/36158/18e80f1f-3775-4457-a2a2-ce3915b5b6c7 width=400>

Before:
<img src=https://github.com/MetaMask/metamask-extension/assets/36158/763c333b-17f0-464a-a5b1-33813a14c7df width=600>

After:
<img src=https://github.com/MetaMask/metamask-extension/assets/36158/a9aad236-0a23-42cb-a15a-61a262cba777 width=600>

## **Pre-merge author checklist**


## **Pre-merge reviewer checklist**
